library/ecp.c

```diff
@@ -20,7 +20,7 @@
 /*
  * References:
  *
- * SEC1 http://www.secg.org/index.php?action=secg,docs_secg
+ * SEC1 https://www.secg.org/sec1-v2.pdf
```

Comment:
How about change this paragraph like this? And add like for RFC 4492 and RFC 7748?
```suggestion
 * - SEC1: https://www.secg.org/sec1-v2.pdf
```

Reply:
Currently prefer to keep it align with the exist comments. 
Anyway, agree that we should re-org the description here, maybe I will raise another PR to deal with that: add links, change comment paragraph,etc.